src/lib.rs

```diff
@@ -75,28 +77,27 @@ impl DeviceManager {
             }
         });
 
+        let telemetry_default_config = vec![crate::telemetry::TelemetryInterfaceConfig {
+            interface_name: "io.edgehog.devicemanager.SystemStatus".to_owned(),
+            enabled: true,
+            period: 10,
+        }];
```

Comment:
The telemetry configuration should be passed as an option of the `new` DeviceManager method.
It should be part of one of the `Runtime` configurations.